3.py

```diff
@@ -0,0 +1,16 @@
+# Напишите программу, которая принимает на вход координаты точки (X и Y), причём X ≠ 0 и Y ≠ 0 и выдаёт номер четверти плоскости, 
+# в которой находится эта точка (или на какой оси она находится).
+# x=34; y=-30 -> 4
+# x=2; y=4-> 1
+# x=-34; y=-30 -> 3
+
+x = int(input('input x: '))
+y = int(input('input y: '))
+if x > 0 and y > 0:
```

Comment:
Это и остальные условия можно свернуть относительно `0`
```suggestion
if x > 0 < y:
```